site/_resources/philosophy.md

```diff
@@ -51,17 +52,23 @@ For example, many projects allow network buffer sizes to be configured arbitrari
 In providing the tools, but little or no guidance on how they should be used, infinitely tweakable knobs forces the burden of mastering the high dimensional configuration space onto the end users of the product.
 The secondary impact on support teams and the upstream development team should not be discounted, both in terms of difficulty in diagnosing problems, and the possibility of configuration drift between environments.
 
-Contour takes the position that when there is a sensible default value for an Envoy configuration parameters, Contour will apply it unconditionally.
-We’ve used this position in the past by compressing HTTP response bodies unconditionally, disallowing TLS/1.0, choosing an aggressive cipher suite, and so on.
+Contour takes the position that when there is a sensible default value for an Envoy configuration parameters, Contour will apply it.
+We've used this position in the past by compressing HTTP response bodies unconditionally, disallowing TLS/1.0 and 1.1, choosing as secure as possible a default cipher suite, and so on.
 
-Contour will provide a method for the administrator or developer to supply their chosen value as a last resort, in the event that we cannot discover a universally acceptable value.
-In the history of the project this second scenario has rarely occurred.
-More often than not, the discussion sparked by the desire to change a particular parameter has led to a deeper understanding of the ways in which Contour is being used which would not have otherwise occurred.
+When Contour was first developed, generally we stopped there.
+However, users of Contour found that this approach, while excellent for the combined cluster operator/application developer users, was not ideal when the cluster operator had to keep a large production deployment up and running.
+
+Contour still and always will attempt to choose the best default for the largest set of users, but we will allow tweaking of those defaults when requested.
+The TLS cipher suite is a great example of this.
```

Comment:
I don't think we need the TLS cipher suite example here, the paragraph before should be enough to keep the doc smaller.

Reply:
Sure, sounds good.